.vscode/tasks.json

```diff
@@ -32,6 +32,12 @@
       "type": "shell",
       "command": "EXCLUDE_CONTRIB=true tools/vscode/refresh_compdb.sh",
       "problemMatcher": []
+    },
+    {
+      "label": "Local Fix Format",
+      "type": "shell",
+      "command": "tools/vscode/local_fix_format.sh -all",
```

Comment:
i dont think this exists

Reply:
wrt local_fix_format im also a bit reluctant to hook this up officially as its not necessarily the best path to achieve this, but wont block - i dont use vscode myself

cc @ravenblackx @jmarantz

Reply:
my bad, write the wrong path.